whois-api/src/main/java/net/ripe/db/whois/api/TopLevelFilter.java

```diff
@@ -0,0 +1,50 @@
+package net.ripe.db.whois.api;
+
+import com.google.common.collect.Maps;
+import net.ripe.db.whois.common.domain.CIString;
+import net.ripe.db.whois.common.etree.NestedIntervalMap;
+import net.ripe.db.whois.common.ip.Interval;
+import net.ripe.db.whois.common.ip.IpInterval;
+import net.ripe.db.whois.common.ip.Ipv4Resource;
+import net.ripe.db.whois.common.ip.Ipv6Resource;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TopLevelFilter<T extends Interval<T>> {
+    private static final HashMap<ObjectType, TopLevelFilter> resourceFilterMap;
```

Comment:
There should be no need to filter out 0/0 or ::/0 as they are not referenced by the organisation entity